src/templates/team.html

```diff
@@ -111,9 +111,15 @@ <h6><label for='cancelEventTextarea'>{{ 'Add a custom message'|trans }}</label><
 <div data-tabcontent='1' hidden>
   <div class='d-flex mb-2'>
     <div id='selectBookableWarningDiv' hidden>{{ 'Select a resource below in order to book it.'|trans }}</div>
-    <div class='ml-auto'>
+    <div class='ml-auto row'>
```

Comment:
remove the `row`, or if it was added for a reason, fix the misalignement of the scope button on the right